docs/appendix/glossary/api-namespace-triplet.md

```diff
@@ -0,0 +1,16 @@
+---
+title: API Namespace Triplet
+id: api-namespace-triplet
+full_link:
+short_description: namespace:type:subtype
+aka:
+---
+
+Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API) to describe how you can interact with that resource.
+
+Each Viam resource's API is uniquely namespaced as a colon-delimited-triplet in the form of `namespace:type:subtype`.
```

Comment:
can we avoid passive voice to make it clearer for the user what being namespaced means (here and in other instances)

Reply:
I might be wrong here and this is how people generally speak about it - In cases like this I generally google a phrase like "namespaced component" or "namespaced object" and see how often it comes up. In this case there a few thousand instances. "an object in the namespace" or defined by the namespace is I think what I would say? @mcvella do you have opinions on this?

Reply:
```suggestion
 These APIs are organized by colon-delimited-triplet identifiers, in the form of `namespace:type:subtype`.
 
 The `namespace` for built-in Viam resources is `rdk`, while the `type` is `component` or `service`.
 `subtype` refers to a specific component or service, like a `camera` or `vision`.

One subtype can have various models, custom or built-in, but they all must conform to the subtype's API definition.
This requirement ensures that when a resource of that model is deployed, you can [interface with it](/program/) using the same [client API methods](/program/apis/) you would when programming resources of the same subtype with a different model.
```
passive voice suggestion at top
extra sentences are likely overboard but could be helpful info?

Reply:
@npentrel @mcvella

Reply:
I like Sierra's suggested change above, its verbose but in a helpful way

Reply:
looks good to me